frontend/talentsearch/src/js/components/pool/PoolAdvertisementPage.tsx

```diff
@@ -269,7 +269,7 @@ const PoolAdvertisement = ({ poolAdvertisement }: PoolAdvertisementProps) => {
             </TableOfContents.Heading>
             <Accordion
               title={intl.formatMessage({
-                defaultMessage: "What are pool recruitment's?",
+                defaultMessage: "What are pool recruitments?",
                 description:
                   "Title for according describing pool recruitment's",
```

Comment:
@esizer Don't forget about the description and its fr.json equivalent where the id will change!